test/test-enotsup-apis.c

```diff
@@ -2,11 +2,6 @@
 #include "uvwasi.h"
 
 int main(void) {
-  /* TODO(cjihrig): This test is intended to be temporary. */
```

Comment:
I guess we should either remove this file now or leave a new comment that this test is intended to verify the behavior of unimplemented APIs but currently there are none.

Reply:
Removed